lib/orogen/gen/project.rb

```diff
@@ -676,14 +680,23 @@ def generate
                 Generation.save_automatic "orogen-project-#{name}.pc.in", pc
 
 		if !self_tasks.empty?
+                    
 		    self_tasks.each { |t| t.generate }
 
 		    deployer = Generation.render_template "tasks", "DeployerComponent.cpp", binding
 		    Generation.save_automatic "tasks", "DeployerComponent.cpp", deployer
 		    pc = Generation.render_template "tasks", "tasks.pc", binding
 		    Generation.save_automatic "tasks", "#{name}-tasks.pc.in", pc
+
+                    if(Orocos::Generation.cpp_proxies?)
```

Comment:
Please follow the ruby style from the rest of the code:
 - no () for if arguments
 - no () for method calls without arguments.
 - definitely no semicolon at the end of the line